src/components/Avatar/Avatar.tsx

```diff
@@ -37,8 +54,10 @@ export function Avatar({size = 'md', variant = 'primary', show, className, ...pr
 
   return (
     <span className="inline-block relative">
-      <img className={classes} {...props} />
+      <img alt="avatar" className={classes} {...props} />
```

Comment:
The `alt` could be empty because the avatar can appear next to the persons name. It is better to not strictly enforce this.